cEP-0005.md

```diff
@@ -0,0 +1,157 @@
+# coala Configuration
```

Comment:
we could choose a common IL format (like a dict the json module would give us) and convert any settings file into this format.